playground/src/components/entry-point/entry-point.spec.js

```diff
@@ -1,13 +1,56 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { MC_API_PROXY_TARGETS } from '@commercetools-frontend/constants';
+import { renderAppWithRedux } from '@commercetools-frontend/application-shell/test-utils';
 import { ApplicationStateMachines } from './entry-point';
 
-describe('rendering', () => {
-  let wrapper;
-  beforeEach(() => {
-    wrapper = shallow(<ApplicationStateMachines />);
+const createStateMachinesListSdkMock = () => ({
+  action: {
+    type: 'SDK',
+    payload: {
+      method: 'GET',
+      service: 'states',
+      options: {
+        perPage: 25,
+        page: 1,
+      },
+      mcApiProxyTarget: MC_API_PROXY_TARGETS.COMMERCETOOLS_PLATFORM,
+    },
+  },
+  response: {
+    count: 25,
+    offset: 0,
+    total: 2,
+    results: [],
+  },
+});
+
+const render = (options) => {
+  return renderAppWithRedux(<ApplicationStateMachines />, {
+    sdkMocks: [createStateMachinesListSdkMock()],
```

Comment:
Have you tried using msw?

Reply:
ah good call.
actually no, I kept it minimal this time.
I can give it a try.

Reply:
I guess it's more of a question what best practices we want to teach our users.

For me something like msw is definitely easier to set up and use, compared to apollo/sdk mocks.

Together with our test-data initiative, I assume that this is the direction we are going to.